src/_actions/checkedFilters.actions.js

```diff
@@ -0,0 +1,22 @@
+import constants from '../_constants';
+
+// constants
```

Comment:
Redundant comment - of course, we already know that you are getting the constant even without the comment - please delete it